lms/templates/edxnotes/note-item.underscore

```diff
@@ -4,9 +4,9 @@
         <p class="note-excerpt-p"><%- message %>
           <% if (show_link) { %>
           <% if (is_expanded) { %>
-              <a href="#" class="note-excerpt-more-link"><%- gettext('Less') %></a>
+              <button type="button" class="note-excerpt-more-link"><%- gettext('Less') %></button>
```

Comment:
Having difficulty getting student notes running locally. Documentation seems out-of-date. Without seeing it, I'm __hopeful__ that because there's a class here, the button is looking like it's supposed to. Looking at the styles in `_student-notes.scss` I feel 75% confident that this should look OK. I will verify on the sandy.